mit_frontend/src/App.css

```diff
@@ -45,7 +45,19 @@ p {
 .user-info-card {
   margin: auto;
   width: 432px;
-  height: 880px;
+  height: 1320px; /*used to be 880px*/
```

Comment:
Just observation here (not sure if this applies, will test in a bit) - it seemed like you had to increase `.App`'s height to `145vh` because the height of `.user-info-card` increased by around 1.5 times - maybe we don't need the height restriction for `.App`?

Reply:
Once I select city with the search result. The NEXT button is outside of the user-info-card. The height should be dynamic once there are more contents generated on the card.

Reply:
+1 for @BruceWeng 's comment, just took a look and we should make this dynamic. Let's remove `height` from both `.user-info-card` and `.App` and supply `padding-bottom` or something to both so the spacing looks appropriate.